wiki/_Footer.md

```
@@ -1,4 +1,4 @@
-**Ultimate** — v1.2.0 `As of 2026-08`. Stable API, semver from here. MIT licensed.
+**Ultimate** — v2.0.0 `As of 2026-08`. Stable API, semver from here. MIT licensed.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)